.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on: 
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Ruby ${{ matrix.version }}, ${{ matrix.gemfile }}
+    strategy:
+      matrix:
+        version: [2.4, 2.5, 2.6, 2.7]
+        gemfile: [Gemfile_ar41, Gemfile_ar50, Gemfile_ar51, Gemfile_ar_master]
+        exclude:
+          - version: 2.7
+            gemfile: Gemfile_ar41
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby ${{ matrix.version }}
+        uses: ruby/setup-ruby@v1
+        with: 
+          ruby-version: ${{ matrix.version }}
+      - name: Install dependencies
+        run: |
+          export BUNDLE_GEMFILE=${GITHUB_WORKSPACE}/${{ matrix.gemfile }}
+          rm -f ${GITHUB_WORKSPACE}/Gemfile.lock
+          gem update --system
```

Comment:
Could we assume that the `ruby/setup-ruby` action will be kept up to date, so we wouldn't have to update rubygems ourselves?